cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -75,15 +77,20 @@ func runApplyAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting IP: %s", err.Error()))
 	}
 
-	ocPath := minishiftConfig.InstanceConfig.OcPath
 	routingSuffix := viper.GetString(routingSuffix)
-	kubeConfigPath := constants.KubeConfigPath
 	sshCommander := provision.GenericSSHCommander{Driver: host.Driver}
+	ocRunner, err := oc.NewOcRunner(minishiftConfig.InstanceConfig.OcPath, constants.KubeConfigPath)
```

Comment:
It should be now. I think historically the "stub" OcRunner in the integration test was there first. The fully fleshed out runner as part of the production code is not around for so long. So yes, it could/should be used in the integration tests.